packages/salesforcedx-utils/package.json

```diff
@@ -3,7 +3,6 @@
   "displayName": "Node Utilities for VS Code",
   "version": "58.15.0",
   "description": "Provides node specific utilities for the Salesforce VSCode extensions",
-  "aiKey": "ec3632a4-df47-47a4-98dc-8134cacbaf7e",
```

Comment:
Keep the value in the package.json.

Reply:
Ok looking at this again. I think I might have mislead you with the comments above. Per our slack discussion I think you're on the right path. Sorry about the mix up.